docs/MakeRelease.html

```diff
@@ -116,7 +116,7 @@ <h2>Procedure used for release 0.1.0</h2>
 git clone https://github.com/chanzuckerberg/shasta.git
```

Comment:
This entire section was the old procedure used for release 0.1.0 which is obsolete and no longer applies. It is time to remove this entire section.

Reply:
Also, remove the previous header "Current procedure". This file is for our use only anyway and it is not linked anywhere in the rest of the documentation.